exercises/practice/kindergarten-garden/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.o
+tests
```

Comment:
These are not necessary, as we have these ignored in the project root.  I would remove this `.gitignore` file unless or until there is something "custom" specifically needed for an exercise.

Reply:
I assumed they are for the benefit of students who who git to manage their solution attempts on their local machines. 

https://github.com/exercism/rust/pull/353
mentions "I like to keep my Exercism solutions backed up on GitHub (e.g. https://github.com/attilahorvath/exercism-rust) and I've seen others do the same."


 The following tracks currently use per-exercise .gitignore:

- elixir
- gleam
- javascript
- purescript
- rust
- swift
- x86-64-assembly


I don't have a preference either way. They were there when I started adding x86-64-assembly exercises, and I followed the existing convention.

Reply:
Nothing here is a critique against your response.  Thank you for that.  Here is my reasoning...

The `.gitignore` files are getting delivered for the exercises, which have have nothing to do with git.  Sounds like "personal choice" to me, and additional information that is not really relevant to the exercises.

A slippery slope, for that reason, as "I assume they are for the benefit of students using X editor" would also justify adding editor related things not only to that `.gitignore` file, but also delivering it with those things in there.

I would argue that us delivering it per exercise causes more student maintenance work, as well, and they should manage their own repositories if that is what they are doing.

I am not against having a documentation thing that describes how they can do such things, and recommended things to ignore though.  But that is a one place area to maintain, and specifically for the track, rather than per exercise.

Reply:
Ack. I have removed the .gitignore file from the current PR. Note I also have a handful of other exercise-adding PRs uploaded for review.  After they get reviewed/merged, I'll delete all of the track's per-exercise  .gitignore files.